star-dash/star-dash/GameEngine/Components/BuffComponent.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 class BuffComponent: Component {
-    var speedMultiplier: CGFloat = 1
+    var speedMultiplier: CGFloat = 1.5
```

Comment:
Why does BuffComponent start with a speed multiplier of 1.5?

Reply:
oops accident